style.css

```diff
@@ -310,3 +310,312 @@ textarea {
 .btn-1:disabled {
   background-color: #2a2b33;
 }
+
+.desktop-menu {
+  display: none;
+}
+
+.textarea {
+  width: 80%;
+  margin-top: 10px;
+}
+
+.recent-work {
+  text-align: center;
+  color: #7d73fc;
+  font-size: 38px;
+  margin-top: 35px;
+  font-family: Roboto, sans-serif;
+}
+
+@media screen and (min-width: 992px) {
```

Comment:
- [ ] You should use min-width of `768px` as per the project requirement
![min-widht-768](https://user-images.githubusercontent.com/44057863/165720455-fad1667f-9b68-4672-b0d0-e7215f952db7.png)
.